src/ranged.cpp

```diff
@@ -2202,9 +2205,12 @@ void target_ui::init_window_and_input()
         ctxt.register_action( "LEVEL_UP" );
         ctxt.register_action( "LEVEL_DOWN" );
     }
-    if( mode == TargetMode::Fire || mode == TargetMode::TurretManual ) {
+    if( mode == TargetMode::Fire || mode == TargetMode::TurretManual || ( mode == TargetMode::Reach &&
+            relevant && relevant->is_gun() && you->get_aim_types( *relevant ).size() > 1 ) ) {
```

Comment:
Check for `relevant != nullptr` is unnecessary here
```suggestion
            relevant->is_gun() && you->get_aim_types( *relevant ).size() > 1 ) ) {
```

Reply:
Why?

Reply:
Because every mode except `TargetMode::Turrets` and `TargetMode::Spell` guarantees `relevant` is not `nullptr`

Reply:
fair enough, i ll make sure to test it in a bit (or at worst tomorrow)